engines/private/private.cpp

```diff
@@ -179,12 +207,12 @@ Common::Error PrivateEngine::run() {
 	Common::Event event;
 	Common::Point mousePos;
 	_videoDecoder = nullptr;
-
+	_language = ConfMan.get("language");
```

Comment:
I would recommend using here `parseLanguage(ConfMan.get("language"))` and then use the language enum instead of string comparison

Reply:
Done.